mjpc/tasks/humanoid/skateboard/steering-task.xml

```diff
@@ -6,17 +6,17 @@
 
   <custom>
     <text name="task_transition" data="Steer" />
-    <numeric name="sampling_representation" data="2" />
     <numeric name="agent_planner" data="2" />
-    <numeric name="agent_horizon" data="0.5" />
-    <numeric name="agent_timestep" data="0.005" />
-    <numeric name="sampling_spline_points" data="16" />
-    <numeric name="sampling_exploration" data="0.15" />
-    <numeric name="sampling_trajectories" data="32" />
+    <numeric name="agent_horizon" data="0.45" />
+    <numeric name="agent_integrator" data="3" />
```

Comment:
This is `implicitfast`, which should be a bit more stable than the default one. If you ever change this, you gotta also change the one in `option` below.